nuxeo-web-ui-ftest/test/features/step_definitions/user_settings.js

```diff
@@ -70,8 +70,8 @@ module.exports = function () {
   });
 
   this.Then(/^I can only see (\d+) authorized application[s]?$/, (numberOfApps) => {
-    this.ui.userAuthorizedApps.waitForVisible();
-    const apps = this.ui.userAuthorizedApps.getApps();
+    const a = this.ui.userAuthorizedApps;
```

Comment:
Do we need this variable?

Reply:
No. It was just used for debugging. Thanks, will remove.